package.json

```diff
@@ -94,7 +94,8 @@
     "socksjs": "^0.5.0",
     "toml": "^2.3.3",
     "uuid": "^3.0.1",
-    "vertica": "0.5.x"
+    "vertica": "0.5.x",
+    "sql-formatter": "latest"
```

Comment:
Could we move sql-formatter to the dev-dependencies? front-end packages are only needed at build time.

You can do this with npm by running this in command line
```sh
npm uninstall sql-formatter
npm install sql-formatter --save-dev
```

Reply:
great appreciate for the command line!

Reply:
Could you remove `"sql-formatter": "latest"` (line 98) since it is in devDependencies now? 
It doesn't look like `npm uninstall sql-formatter` removed it.